src/ontology/uberon-edit.obo

```diff
@@ -224330,6 +224330,20 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
+name: transformation zone of the cervix
+def: "Site of transition between the stratified squamous epithelium of the ectocervix and the mucus-secreting simple columnar epithelium of the endocervix. It is adjacent to the squamocolumnar junction. Before puberty and after menopause the transformation zone resides in the cervical canal. During the reproductive age of the woman is located just outside the external os. Squamous cervical cancer originates in the transition zone. Glandular cervical cancer originates in either the transformation zone or the glandular epithelium above the transformation zone." [https://www.ncbi.nlm.nih.gov/books/NBK568392/, ISBN:0781750563]
```

Comment:
Regarding the text definition:

- Cannot access ISBN text, so cannot crosscheck that

- Consider the following text definition, which specifically associates this class with "epithelium", which seems reasonable since it is a subclass of 'epithelium':
"The area of **epithelium** that lies between the native and unaffected columnar epithelium of the endocervical canal and the native squamous epithelium deriving from the vaginal and ectocervical squamous epithelium." https://www.ncbi.nlm.nih.gov/books/NBK568392/

- Move the rest of the current text to a comment and add missing word ("it"):
"... During the reproductive age of the woman **it** is located ..."